multipath/multipath.conf.5

```diff
@@ -1168,10 +1168,10 @@ listed in the \fBwwids_file\fR. Users can manually set up multipath maps using t
 \fBmultipathd add map\fR command. Once set up manually, the map is
 remembered in the wwids file and will be set up automatically in the future.
 .TP
-.I no
+.I off|no
```

Comment:
I find the vertical bar syntax confusing here. We use it nowhere else in the man page. Can you just put "off", and add "'no' is an alias for 'off'" further down?